### PR TITLE
Initial SQL 4 CDS extension listing

### PR DESCRIPTION
**Pull request by MarkMpn**
Added listing for SQL 4 CDS extension to let users query data in Microsoft Dataverse/Dynamics 365 instances from Azure Data Studio.

I've included both a direct link to the vsix file and to the GitHub release page for the downloads, please let me know if I need to remove one of them.

More information about the extension is available at https://markcarrington.dev/2023/01/31/sql-4-cds-v7-1-released/#azure_data_studio

### CODE REVIEW

File: extensionsGallery.json [hunk 1]
```diff
@@ -4828,6 +4828,75 @@
...
+						{
+							"version": "7.1.0",
+							"lastUpdated": "02/01/2023",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+									"source": "https://github.com/MarkMpn/Sql4Cds/releases/download/v7.1.0/azuredatastudio-sql4cds-7.1.0.vsix"
+								},
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:4852]: Unfortunately, at this time we can only provide direct VSIX links to extensions we own. All 3rd party extensions need to use the `Microsoft.SQLOps.DownloadPage` asset type with a link to the download page only.


```suggestion
```

File: extensionsGallery.json [hunk 2]
```diff
@@ -4828,6 +4828,75 @@
...
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
+									"source": "https://github.com/MarkMpn/Sql4Cds/releases/tag/v7.1.0"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+									"source": "https://markcarrington.dev/wp-content/uploads/2019/10/SQL4CDS-Icon.png"
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:4859]: All resource files will need to be hosted on github (and the raw.githubusercontent link used as the URL) currently

File: extensionsGallery.json [hunk 3]
```diff
@@ -4828,6 +4828,75 @@
...
+									"key": "Microsoft.VisualStudio.Code.Engine",
+									"value": "^1.65.2"
+								},
+								{
+									"key": "Microsoft.AzDataEngine",
+									"value": "*"
+								},
+								{
+									"key": "Microsoft.VisualStudio.Services.Links.Source",
+									"value": "https://github.com/mongodb-js/vscode"
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:4893]: This doesn't look correct

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]
Please also: 

1. Add this to the [insiders gallery](https://github.com/microsoft/azuredatastudio/blob/release/extensions/extensionsGallery-insider.json) as well
2. Increase the extension count at the bottom of the file

File: extensionsGallery.json (hunk 3 above)
Comment (MarkMpn) [reply extensionsGallery.json]: Sorry, copy & paste error! Fixed in the latest update.

File: extensionsGallery.json (hunk 2 above)
Comment (MarkMpn) [reply extensionsGallery.json]: Thanks, I've updated this with a raw.githubcontent.com URL now.

File: extensionsGallery.json (hunk 1 above)
Comment (MarkMpn) [reply extensionsGallery.json]: OK, I've removed the vsix link and just left the URL to the GitHub release page for downloads.

**Comment by Charles-Gagnon** [comment]
And it's up, thank you for your contribution!

image

**Comment by MarkMpn** [comment]
Thank you! Do you have any instructions on how to update the List of Extensions wiki page, as the link from the bot 404s?

**Comment by Charles-Gagnon** [comment]
That link for our internal usage - I've went ahead and updated the page with your extension!

**Pull request merged**